docs/internals/howto-release-django.txt

```diff
@@ -52,6 +52,17 @@ Prerequisites
 
 You'll need a few things before getting started:
 
+* A Unix environment with the following tools installed (in alphabetical order):
+
+  * bash
+  * git
+  * GPG
+  * make
+  * man
+  * hashing tools (``md5sum``, ``sha1sum``, ``sha256sum``)
+  * python
+  * ssh
+
 * A GPG key. If the key you want to use is not your default signing key, you'll
```

Comment:
Do we want to mention that this key needs to be uploaded to GitHub and in Jenkins?